server/index.js

```diff
@@ -43,6 +46,13 @@ if (isDev) {
     app.use(require('errorhandler')());
 }
 
+function errorHandler(err, req, res, next) {
+    console.log('errorHandler');
+    res.status(500);
+    render(req, res, { view: '500' });
+    next(err); // added this row because eslint says: error  'next' is defined but never used  no-unused-vars
```

Comment:
just put the function inside special comments:
```js
/*eslint-disable no-unused-vars */
...
/*eslint-enable no-alert */
```

for more info see http://stackoverflow.com/questions/27732209/turning-off-eslint-rule-for-a-specific-line